src/types/i18n.d.ts

```diff
@@ -0,0 +1,1654 @@
+declare module 'I18n/en-US.json' {
```

Comment:
an important file

Reply:
i've been thinking about creating a normal `.ts` file, but IMO `.d.ts` is fine in this case